rllib/policy/policy.py

```diff
@@ -1494,6 +1494,17 @@ def _initialize_loss_from_dummy_batch(
                 stats_fn(self, train_batch)
             if hasattr(self, "stats_fn") and not self.config["in_evaluation"]:
                 self.stats_fn(train_batch)
+        else:
+            # This is not needed to run a training with the Learner API, but useful if
+            # we want to create a batche of data for training from view requirements.
```

Comment:
typo: batche

Curious to know if this was required at all? What problem is this solving?